### DISCUSSION TITLE
Please add ability to move control buttons to left side in resource table

### DISCUSSION THREAD

**Discussion by s-fog** [laravel/nova-issues#5088] [Ideas & Feature Requests]
Can u do it? Because when table have a lot columns, it is really hard to use this buttons. I have several like that tables.

**Comment by s-fog** [comment]
guys?

**Comment by Tarpsvo** [comment]
Upvoted as I would also deem this a worthy feature. Some advanced usecases require long tables, some of which should only be editable and not viewable. So I would be grateful if there was an option of:
1) customizing the location of the buttons (left/right)
2) customizing the visibility of the buttons (ie hiding the "view" button, as clicking on the ID already does the same thing)

**Reply by s-fog** [reply]
1+

**Comment by vbezruchkin** [comment]
One more vote here!

We have pretty long lists with the required information. It fits most of our screens, BUT it does not fit some screens lower than 1920 (for example). In this case, it would be preferable to have control buttons first, and then the information.

There is a package for this, but it would be nice to have it in the box. A configuration (like we have for pagination) would be super useful! thanks for the consideration!

**Comment by vesper8** [comment]
Been asking for this feature for 7+ years... I don't think it'll ever come :(

But that's never stopped me. I was using 

https://github.com/Flagstudio/nova-actions-left on Nova 3

then

https://github.com/saintsystems/nova-actions-left on Nova 4

and now

https://github.com/vesper8/nova-actions-left on Nova 5

 : )
